rpm/sailfish-components-webview.spec

```diff
@@ -135,6 +136,8 @@ Unit tests and functional tests of Sailfish OS WebView / WebEngine
 
 %prep
 %setup -q -n %{name}-%{version}
+sed -i "s/^Version: .*/Version: "$(echo %{version} | sed 's/+.*//')"/" \
+    ./lib/pkgconfig/sailfishwebengine.pc
```

Comment:
Our integration system takes the tag from the git and uses that for the rpm package. That's the reason why we're "always" passing the VERSION from the spec down to the qmake (pc file generation). A bit depending on how are you building this could help you (SailfishOS-latest being the target name):

Use precisely version from the spec
> mb2 -t SailfishOS-latest -X build

Then without -X you get latest tag + extra identifier:
> mb2 -t Sailfish-latest build

Reply:
If you want to get this merged please drop this seding part.

This is enough:
```
%qmake5 VERSION=%{version}
```

Please rebase as well.